Gaijinizer patch/Map230.txt

```diff
@@ -4,124 +4,137 @@
+\\N[1]! What was with that old man!
+
+I couldn't relax because he kept looking in my
+direction every time!#Strate
 > END STRING
 
 > BEGIN STRING
 エジプトの考古学者さんだよ……
 
 今は、ピラミッド周りの遺跡の発掘をやってるみたい。
 > CONTEXT: Map230/events/1/pages/0/18/Dialogue/kazuya
-I am an archeologist from Egypt ...
-Now, I'm going to excavate the ruins around the
-pyramid.#MTLed
+He's an archeologist from Egypt......
+
+It seems he is currently excavating the ruins
+around the pyramid.#Strate
 > END STRING
 
 > BEGIN STRING
 結局、Ｍ.Ｏ.Ｗの関与している可能性は低いって事で
 
 依頼は受けないことになったけどね……
 > CONTEXT: Map230/events/1/pages/0/22/Dialogue/kazuya
-After all, it is unlikely that M.O.W. is involved
-I have not received the request, though ...#MTLed
+In the end, I didn't accept the job, because it
+was unlikely that M.O.W. was involved......#Strate
 > END STRING
 
 > BEGIN STRING
 ……………ねぇ\\N[1]。
 
 ボクって、そんなにアホ面かなぁ…？
 > CONTEXT: Map230/events/1/pages/0/28/Dialogue/enemy11
-................... Hey \\N[1].
-I am so stupid ...?#MTLed
+......Hey, \\N[1].
+
+Do I really look that stupid...?#StrateLocalization
 > END STRING
 
 > BEGIN STRING
 うん。
 > CONTEXT: Map230/events/1/pages/0/32/Dialogue/kazuya
-Yup.#MTLed
+Yup.#Strate
 > END STRING
 
 > BEGIN STRING
 \\{即答しなくてもよくない！？
 > CONTEXT: Map230/events/1/pages/0/34/Dialogue/enemy12
-\\{I do not have to answer immediately!
-?#MTLed
+\\{You don't have to answer
+immediately!?#Strate
 > END STRING
 
 > BEGIN STRING
 だって……実際謎解き得意じゃないじゃん……
 
 『例の問題』以外、ホント小学生レベルだし……
 > CONTEXT: Map230/events/1/pages/0/36/Dialogue/kazuya
-Because...not really good at solving the
-mystery...... It is true elementary school
-student level except "problem of example" ...#MTLed
+After all...... You're not really good at solving
+riddles...... Besides your "usual riddle", you're
+really on the level of an elementary schooler...#Strate
 > END STRING
 
 > BEGIN STRING
 こないだの新作ホラーゲームだって
 
 謎解き部分が分からなくて投げてたじゃん……
 > CONTEXT: Map230/events/1/pages/0/40/Dialogue/kazuya
-It's a new horror game. I was throwing it because
-I did not know the mystery solving part ...#MTLed
+You even threw out that new horror game because
+you didn't understand the puzzles......#StrateLocalization
 > END STRING
 
 > BEGIN STRING
 だってぇ……
 
 謎解きの鍵が同じ部屋にないと詰む性質だもん、ボク…
 > CONTEXT: Map230/events/1/pages/0/44/Dialogue/enemy11
-Because......... It is the nature that the key
-to solve the mystery is not in the same room.#MTLed
+That's because.........
+
+If the key to solving the puzzle is not in the
+same room, I get completely lost...#Strate
 > END STRING
 
 > BEGIN STRING
 それに、いくら歴史を感じる装置だからってさぁ
 
 変な命令口調で分かりにくく書く必要なくないー？
 > CONTEXT: Map230/events/1/pages/0/48/Dialogue/enemy11
-Besides, how much the device feels history
-You do not need to write in a weird command tone
-and make it difficult to understand?#MTLed
+Besides, no matter how historic the game feels,
+does it really need to write everything in that
+weird commanding tone that makes it so difficult
+to understand?#StrateUnsure
 > END STRING
```

Comment:
Ron Note

Reply:
That first line could be rewritten to something like:

"Besides, just because it's got a really historical
setting..."

Or even completely rearranged to:

"Besides, does it really need to write everything
in that weird, hard-to-understand commanding
tone just because it's in a historical setting?"

But otherwise no translation issues.

Reply:
I like the latter version you said, let's go with that.